README.md

```diff
@@ -19,7 +19,7 @@ delegation mechanism:
   enabling tight control over which third party origins can access requested
   hint data.
 
-The goal of Client Hints is to **reduce active fingerprinting** on the web while
+The goal of Client Hints is to **reduce passive fingerprinting** on the web while
```

Comment:
Thanks for catching that!!

Reply:
LOL I had a whole explanation of how this could be so worked out in my head...